Algorithms.cpp

```diff
@@ -29,7 +35,9 @@ int main()
     int input, numb, lo = 0, hi = sizeof(arr)/sizeof(arr[0]);
 
     cout << "Which algorithm would you like to run? Enter an integer:" << endl;
-    cout << "1. linear search,   2. binary search (requires ordered array)" << endl;
+    cout << "1. linear search,   2. binary search (requires ordered array)" 
+        << endl;
```

Comment:
why is this endl on a new line but the endl of the line above is on the same line?